### ISSUE TITLE
Problem pasting image

### ISSUE THREAD

**Issue by mancda** [MicrosoftEdge/WebView2Feedback#2900]
I have upgraded my UWP App to use webview2 and when I navigate to a online M365 Word document I am now unable to paste an image from the clipboard.

Thanks for your help

**Comment by nishitha-burman** [comment]
Hi @mancda,

Can you please provide more details about your issue?

**Version**
SDK: 
Runtime: 
Framework: 
OS: 

**Repro Steps**

**Comment by mancda** [comment]
Version: WinUI 2.8.1
SDK: 10.0.22621.0
Runtime:
Framework: .NetCore 6.2.14
OS: Windows 11

Steps to reproduce

1. Create new UWP app
2. Add Webview2 component
3. Navigate to a Online Microsoft 365 Word Document
4. Attempt to paste an image into the document from the clipboard.

**Comment by chrisjshields** [comment]
Same issue here. Cannot paste images onto any site

**Comment by maurawinstanley** [comment]
Thanks for reporting the issue, this is a known limitation that we have tracked on our backlog, I will update the thread when we begin the work!

**Comment by DotNet-Fan** [comment]
@champnic @maurawinstanley  is this related to #3601?

**Comment by champnic** [comment]
Yes - I'll close #3601 as a duplicate. Thanks for making the connection here :)

**Comment by DotNet-Fan** [comment]
@champnic  Any update on this issue?

Thanks

**Comment by champnic** [comment]
No updates here yet.

**Comment by tamasbito99** [comment]
I have the same issue.
@champnic Do you have any information about when will be this bug fixed?

**Comment by jennifer-jacobs** [comment]
Hi @tamasbito99 & @DotNet-Fan @mancda @chrisjshields,
Copying and pasting of images should work in UWP WV2 apps starting with runtime version 131 which should be out around mid-November.

There is still an open bug around copying with the context menu which can still repro rarely (#3775), but ctrl + c and ctrl +v should work reliably. The context menu issue only repros ~10% of the time.

If you continue to have issues let us know!

**Comment by matthiashack** [comment]
After two years still no developer could resolve this

**Comment by jennifer-jacobs** [comment]
Hi @matthiashack - are you using a runtime version 131 or greater? This issue should be fixed in those versions. 

Can you provide steps to reproduce the issue you're still seeing?

**Comment by matthiashack** [comment]
i find the reason is that i wrongly set the params bool contextMenuEnabled = false;  in my code, thank you  @jennifer-jacobs

**Comment by matthiashack** [comment]
![Image](https://github.com/user-attachments/assets/7ee87c35-f534-4fdc-bf77-90a0d6f79b06)
